llvm/include/llvm/IR/IntrinsicsSPIRV.td

```diff
@@ -91,6 +91,7 @@ let TargetPrefix = "spv" in {
   def int_spv_sign : DefaultAttrsIntrinsic<[LLVMScalarOrSameVectorWidth<0, llvm_i32_ty>], [llvm_any_ty], [IntrNoMem]>;
   def int_spv_radians : DefaultAttrsIntrinsic<[LLVMMatchType<0>], [llvm_anyfloat_ty], [IntrNoMem]>;
   def int_spv_group_memory_barrier_with_group_sync : DefaultAttrsIntrinsic<[], [], []>;
+  def int_spv_clip : DefaultAttrsIntrinsic<[], [], []>;
```

Comment:
I don't think we should call this intrinsic clip just because it is used by HLSL clip builtin. Especially since the signature is different from the dx clip. If other folks are ok maybe `int_spv_discard` would be a better name?